content/api-docs/basic-concepts/01-entry.md

```diff
@@ -70,3 +70,34 @@ Binary data (value of a binary field) is represented by an [OData Named Resource
     },
     ...
 ```
+
+# Valid entity urls
+There is an important distinction between an entity path (that adresses a single entity and gets a response containing the _fields of that entity_) and a collection path that gets the _child entities_ of that collection. In this section we list the valid entity requests. For collection requests please see the [next page](/api-docs/basic-concepts/02-collection).
+
+
+The important thing to note is that the content name is enclosed in **parentheses AND single quotes**. This allows you to have content in the repository with a name containing parentheses:
+
+```
+https://dev.demo.sensenet.com/OData.svc/Root/Content('MyWorkspace')         // content name: MyWorkspace
+https://dev.demo.sensenet.com/OData.svc/Root/Content('Folder(1)')           // content name: Folder(1)
+https://dev.demo.sensenet.com/OData.svc/Root/Content('mydocument(2).docx')  // content name: mydocument(2).docx
+https://dev.demo.sensenet.com/OData.svc/Root/Content('42')                  // content name: 42
+https://dev.demo.sensenet.com/OData.svc/Root/Content('(42)')                // content name: (42)
+https://dev.demo.sensenet.com/OData.svc/content(1234)
```

Comment:
maybe add "// content id: (1234)" to emphasize more the difference

Reply:
thanks, done